Fabric.Authorization.API/Modules/UsersModule.cs

```diff
@@ -61,33 +61,47 @@ public class UsersModule : FabricModule<User>
 
         private async Task<dynamic> AddGranularPermissions(dynamic param)
         {
-            var apiModel = this.Bind<GranularPermissionApiModel>();            
+            var apiModel = this.Bind<List<PermissionApiModel>>();            

```

Comment:
After binding the model, should we check to see if the list is empty, and if so, let the consumer know that there were no permissions in the post? I've seen this happen when they try to pass something that is not an array of objects, and the nancy serializer just creates an empty list.

Reply:
yeah, good call